pkg/controller/instance/instance_controller.go

```diff
@@ -240,7 +228,7 @@ func (r *Reconciler) Reconcile(request ctrl.Request) (ctrl.Result, error) {
 	log.Printf("InstanceController: Going to proceed in execution of active plan '%s' on instance %s/%s", activePlan.Name, instance.Namespace, instance.Name)
```

Comment:
Ah, and here it's using "active plan", not "plan in progress". So my nitpick above is also about consistency 🤓